SignalServiceKit/src/Storage/OWSStorage.m

```diff
@@ -332,29 +325,24 @@ - (void)runAsyncRegistrationsWithCompletion:(void (^_Nonnull)(void))completion
     OWS_ABSTRACT_METHOD();
 }
 
-+ (NSArray<OWSStorage *> *)allStorages
++ (void)setupStorageWithMigrationBlock:(OWSStorageMigrationBlock)migrationBlock
```

Comment:
naming nit: `registerExtensions` might be more preices than `setupStorage`

Reply:
Done.